### PR TITLE
Request reviews

### PR DESCRIPTION
**Pull request by fickryiman**
- merge requests from development to the main branch.
- All requirements has fulfilled.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,356 @@
...
+- display a air quality and weather informations based on selected game
+- single page apps (SPA)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+## 🚀 Live Demo <a name=""></a>
+
+<!-- > Add a link to your deployed project. -->
+- Click the following url for [... Live Demo ...](https://free-pc-games.vercel.app/)
```
Comment (VICTOR-CHUKWU) [inline README.md:187]: - [x] I can not see your live video link where you demonstrated how your project works, kindly include a 5 mins mx video presentation of your project

File: src/App.css [hunk 2]
```diff
@@ -0,0 +1,381 @@
...
+.error-message {
+  margin-top: 8rem;
+  font-size: 1.49rem;
+}
+
+.games-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+}
```
Comment (VICTOR-CHUKWU) [inline src/App.css]: - [x] your project is an interesting one and your design is excellent however looking at the given template the home page is a two column grid, but yours is one, kindly make it look like the given design
your design :point_down: 
Screenshot from 2023-05-25 20-40-32
given design :point_down: 
Screenshot from 2023-04-28 11-04-25

File: src/Components/Loader.jsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        color="#4fa94d"
+        wrapperStyle={{}}
+        wrapperClass=""
+        visible={visible}
+        ariaLabel="three-circles-rotating"
+        outerCircleColor="white"
+        innerCircleColor="white"
+        middleCircleColor="white"
+      />
+    </div>
```
Comment (VICTOR-CHUKWU) [inline src/Components/Loader.jsx:19]: - [x] interesting design :clap:

File: src/App.test.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  render(<App />);
+  const linkElement = screen.getByText(/learn react/i);
+  expect(linkElement).toBeInTheDocument();
+});
```
Comment (VICTOR-CHUKWU) [inline src/App.test.js]: - [x] one of your test is failing, kindly fix this
image

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi  @fickryiman , 

## STATUS: CHANGES REQUIRED :recycle: 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/App.css (hunk 2 above)
Comment (fickryiman) [reply src/App.css]: changes mobile version into two template column

File: src/Components/Loader.jsx (hunk 3 above)
Comment (fickryiman) [reply src/Components/Loader.jsx:19]: Thank you

File: src/App.test.js (hunk 4 above)
Comment (fickryiman) [reply src/App.test.js]: fixes integration test, removed test files from create-react-app boilerplate

File: README.md (hunk 1 above)
Comment (fickryiman) [reply README.md:187]: add video demonstrating the project

File: src/App.css (hunk 2 above)
Comment (fickryiman) [reply src/App.css]: mobile-version

File: README.md (hunk 1 above)
Comment (fickryiman) [reply README.md:187]: video presentations

File: src/App.test.js (hunk 4 above)
Comment (fickryiman) [reply src/App.test.js]: integration test free-pc-games

**Review by 1GORDON — COMMENTED** [review]
##
## Hi @fickryiman ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

### This project will be marked as Invalid and won't affect the number of your weekly reviews.

- [x] You have submitted a project with the wrong GitFlow. Ensure your feature branch merges into `dev` or `development` instead of `main`.

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by fickryiman** [comment]
Hi @1GORDON  change to the default main branch only for this review request purpose, previously I worked at the development branch as the default.

**Comment by fickryiman** [comment]
Hi @1GORDON , how do I send a request review if not to the main branch?

File: README.md [hunk 5]
```diff
@@ -1,2 +1,359 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Live Preview
+[![Watch the video](https://cdn.loom.com/sessions/thumbnails/79ba5c7cbecd49a18c746fbf63d735d2-with-play.gif)](https://www.loom.com/embed/79ba5c7cbecd49a18c746fbf63d735d2)
+
```
Comment (abdallahmalima) [inline README.md]: - [x] You have done a great job 💪of recording a video and including it in the readme file📜, as requested by the previous reviewer🙄. However, your video is missing a demonstration of a piece of code that you are proud👍 of. This is a key requirement for the project, and including it in your video will enhance your presentation, meet the project requirements, and demonstrate your effort effectively. 🙏Please review the project requirements below and kindly add this part to your video.
image

**Review by abdallahmalima — CHANGES_REQUESTED** [review]
Hi @fickryiman , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

A little bit more


 ## Highlights
- Project deployed✔
- Correct project structure✔
- `Home` page and `Detail` page

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 5 above)
Comment (fickryiman) [reply README.md]: Add a new video demonstration that highlights the interesting features I am proudly making for.

File: README.md [hunk 6]
```diff
@@ -1,2 +1,359 @@
...
+<!-- LIVE DEMO -->
+## 🚀 Live Demo <a name=""></a>
+
+<!-- > Add a link to your deployed project. -->
+- Click the following url for [... Live Demo ...](https://free-pc-games.vercel.app/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Live Preview
+[![Watch the video](https://cdn.loom.com/sessions/thumbnails/d51acfe30a5748d3a444a81b6bbd5e86-with-play.gif)](https://www.loom.com/embed/d51acfe30a5748d3a444a81b6bbd5e86)
```
Comment (Qoosim) [inline README.md]: - [x] Great job making your presentation video. However, the project requirement about the presentation video hasn't completely met. You're yet to highlight any pieces of code that you're very proud of. This simply means that you're expected to show some line of code in your text editor that responsible for a feature in your project, not to just say it alone while presenting your project in the video. Show some line of code in your vscode text editor and show  the feature that line of code responsible for on your page. Kindly do that.

[Your Video](https://www.loom.com/embed/d51acfe30a5748d3a444a81b6bbd5e86)

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle: 

Hi @fickryiman, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 




 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 6 above)
Comment (fickryiman) [reply README.md]: only have 5 minutes for the presentation and also show some lines of code, ok.

Comment (fickryiman) [reply README.md]: I'll try it

**Review by DammyShittu — APPROVED** [review]
Hi @fickryiman,

## APPROVED 🟢

## Highlights
✔️ Descriptive documentation

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**